guides/RELEASES.md

```diff
@@ -14,7 +13,7 @@ Release process
 * Pull request approved
 * Feature branch merged into `develop`
 * Developer tests feature in `develop` environment
-* Developer moves story from team board to release board `PO sign off` column
+* Developer moves story from team board to `PO sign off` column
```

Comment:
The following could potentially be clearer as the card isn't being moved off of the team board:

```suggestion
* Developer moves story to `PO sign off` column
```